app/(auth)/signup/page.tsx

```diff
@@ -0,0 +1,15 @@
+'use client';
+import SignUpForm from '@/components/SignUpForm';
+
+function Page() {
+  return (
+    <div className="wrapper">
+      <div className="form-wrapper">
+        <h1 className="mt-70 mb-30">Sign up</h1>
+        <SignUpForm></SignUpForm>
```

Comment:
If there is no children inside, I would recommend having it done as so: 
`<SignUpForm/>`